doc/nrf/protocols/bt_mesh/dfu_over_ble.rst

```diff
@@ -0,0 +1,72 @@
+.. _dfu_over_ble:
+
+Point-to-point DFU over Bluetooth Low Energy
+############################################
+
+Following the instructions described in :ref:`FOTA over Bluetooth Low Energy<ug_nrf52_developing_ble_fota>`, you can enable the support for and perform FOTA updates using a mobile app.
+
+If the device's composition data is going to change after the FOTA update on a Bluetooth mesh device is performed, unprovision the device before downloading the new image.
+
+If you are using the `nRF Connect Device Manager`_ mobile app to perform FOTA updates, your Bluetooth mesh device might not be visible in the list of available devices.
```

Comment:
Is it intentional that this is repeated in the note below?

Reply:
Looking at it now, it seems like a good idea to remove it. But I remember this being discussed at the time it was added. I do not remember the reason it was written like that, but it was intentional. I will not remove it now, but we can revisit this when Wille and I discuss the user guide.

Reply:
NCSDK-21772